### PR TITLE
Doc fix pip

### PR DESCRIPTION
**Pull request by inakypg**

### CODE REVIEW

**Review by lpereira — DISMISSED** [review]
LGTM, but please also change instructions for Windows and Mac.

File: doc/getting_started/installation_linux.rst [hunk 1]
```diff
@@ -64,15 +64,12 @@ Install the required packages in a Fedora host system with:
...
+	 doxygen dfu-util dtc python3-pip
 
 
 Install additional packages required for development with Zephyr::
 
-   $ curl -O 'https://bootstrap.pypa.io/get-pip.py'
-   $ ./get-pip.py
-   $ rm get-pip.py
-   $ pip3 install -r scripts/requirements.txt
+   $ pip3 install --user -r scripts/requirements.txt
```
Comment (nashif) [inline doc/getting_started/installation_linux.rst]: we need this changed for all OSes

**Review by nashif — DISMISSED** [review]

File: doc/getting_started/installation_linux.rst (hunk 1 above)
Comment (inakypg) [inline doc/getting_started/installation_linux.rst]: Windows and MAC have the concept of --user? how cute...

**Review by dbkinder — DISMISSED** [review]
+1

**Comment by inakypg** [comment]
?? a single TC is failing in QEMU, but this a doc fix -- https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/2652/consolidateTests

File: doc/getting_started/installation_linux.rst [hunk 2]
```diff
@@ -55,7 +55,7 @@ Install the required packages in a Ubuntu host system with:
 .. code-block:: console
 
    $ sudo apt-get install git make gcc g++ ncurses-dev \
-	 doxygen dfu-util device-tree-compiler
+	 doxygen dfu-util device-tree-compiler python-pip3
```
Comment (andrewboie) [inline doc/getting_started/installation_linux.rst]: Are you absolutely sure this is called python-pip3 and not python3-pip?
https://packages.ubuntu.com/search?keywords=python3-pip

**Pull request merged**